docs/node-operators/migration-guides/upgrade-to-0710.md

```diff
@@ -0,0 +1,401 @@
+2. The person responsible for updating genesis needs to create a PR with changes.
+3. All of the validators need to accept changes and approve the PR.
+4. Approved PR must be merged by one of the validators.
+
+:::note
+Make sure, you have correct values for the following parameters in your genesis: `genesis_time`, `time_iota_ms`, `chain_id`, `network_parameters_checkpoint_overwrite`.
+:::
+
+:::note Checkpoint parameters overwrite
+
+All of the network parameters will be taken from checkpoint except the following:
+
+- Added since the previous released version
+- Mentioned in the `network_parameters_checkpoint_overwrite` section
+:::
+
+### 6. Download new genesis file
+After creating a backup and preparing a new genesis file, put it on your server. All the validators **must** use the same genesis file. 
+
+1. Download genesis
+2. Remove old genesis at `<TENDERMINT-HOME>/config/genesis.json`
+3. Save new, downloaded genesis to `<TENDERMINT-HOME>/config/genesis.json`
+4. Verify genesis - see example below
+
+#### Example workflow
+An example workflow for reviewing the genesis file may look like following:
+
+```bash 
+# Download genesis
+wget https://raw.githubusercontent.com/vegaprotocol/networks/master/testnet2/genesis.json
+
+# Move old genesis to a different location
+cp <TENDERMINT-HOME>/config/genesis.json <TENDERMINT-HOME>/config/genesis.json.bk
+
+# Copy genesis to its final location
+cp ./genesis.json <TENDERMINT-HOME>/config/genesis.json
+
+# Verify genesis
+<VEGA-BIN> verify genesis <TENDERMINT-HOME>/config/genesis.json
+```
+
+### 7. Update Vega core config
+
+There is no change required in the `vega core` config. If you have to prepare the `vega` config from scratch, use the instruction for [v0.68.2](./upgrade-to-0682.md#7-update-vega-core-config).
+
+### 8. Update Tendermint config
+
+There is no change required in the `tendermint` config. If you have to prepare the `tendermint` config from scratch, use the instruction for [v0.68.2](./upgrade-to-0682.md#8-update-tendermint-config).
+
+### 9. Migrate tendermint data when using default home path
+
+Vega has migrated from the tendermint to its fork, called the [CometBFT](https://cometbft.com). This change has been introduced in the previous version of the vega. If you are using the default home path for the Tendermint(`~/.tendermint`), we recommend migrating it to the default for the CometBFT(`~/.cometbft`). You can read more about the change and required steps in the [previous migration guide](./upgrade-to-0702.md#9-migrate-tendermint-data-when-using-default-home-path)
+
+### 10. Update data node config
+
+#### The Gateway Port settings
+
+We have merged the `GraphQL` and the `REST` APIs into a single port. Now you must define the `Port` and `IP` in the `[Gateway]` section instead of its child. Remove the `Port` and `IP` from the `[Gateway.GraphQL]` and `[Gateway.REST]` sections.
```

Comment:
```suggestion
#### Gateway port settings

The GraphQL and REST APIs have been merged into a single port. As a result, you now must define the `Port` and `IP` in the `[Gateway]` section instead of its child. Remove the `Port` and `IP` from the `[Gateway.GraphQL]` and `[Gateway.REST]` sections.
```